lib/AST/Type.cpp

```diff
@@ -2447,6 +2447,16 @@ getForeignRepresentable(Type type, ForeignLanguage language,
         break;
       }
     }
+    
+    // Specialize the conformance we were given for the type we're testing.
+    if (result.getKind() == ForeignRepresentableKind::Bridged
```

Comment:
This change looks like it's related to Objective-C bridging, is there a test case for it?

Reply:
I caught it incidentally while trying to build with my `-verify-all-substitution-maps` flag I added. I didn't reduce a test case.

Reply:
Alright, no worries.